test/server/hot_restart_impl_test.cc

```diff
@@ -111,120 +111,5 @@ TEST_F(HotRestartImplTest, crossAlloc) {
   EXPECT_EQ(stat5, stat5_prime);
 }
 
-TEST_F(HotRestartImplTest, truncateKey) {
```

Comment:
I moved a lot of these to `stats_impl.cc`. See the other comment here: https://github.com/ambuc/envoy/pull/1/files#r182548087